applications/plugins/Geomagic/src/Geomagic/GeomagicDriver.cpp

```diff
@@ -490,8 +490,11 @@ void GeomagicDriver::draw(const sofa::core::visual::VisualParams* vparams)
 }
 
 
-void GeomagicDriver::computeBBox(const core::ExecParams*  params, bool  )
+void GeomagicDriver::computeBBox(const core::ExecParams*  params, bool onlyVisible)
 {
+    SOFA_UNUSED(params);
+    SOFA_UNUSED(onlyVisible);
```

Comment:
From what I can see, all the implementations of `computeBBox` have the following code:

```suggestion
    if (!onlyVisible) return;
```